tests/e2e/envfuncsext/nebulacluster-ready-func.go

```diff
@@ -110,6 +113,24 @@ func defaultNebulaClusterReadyFuncForStatus(_ context.Context, _ *envconf.Config
 				)
 			}
 		}
+
```

Comment:
This func is used to cmp nc status, maybe put those in `defaultNebulaClusterReadyFuncForGraphd/Metad/Storaged` is more suitable.

Reply:
ok